_authors/osci_2023/sushanthvutukuri.markdown

```diff
@@ -0,0 +1,8 @@
+---
+name: Sushanth Kumar Vutukuri
+short_name: Sushi
+twitter: NA
```

Comment:
Please remove the `twitter` line and we should be good to go!

Reply:
I believe I edited it correctly so that the new included commits should have the file formatted correctly. If I made a mistake please let me know.